content/ui/admin/init.collage.js

```diff
@@ -156,7 +156,7 @@ GLOB.pers.init = function(){
         });
     
     $.concierge.addComponents({
-        notes_loader:    function(P, cb){GLOB.pers.call("getNotes", P, cb);}, 
+        notes_loader: function(P, cb){GLOB.pers.__components.notes_loader(P, cb);}, 
```

Comment:
I think this line should be removed. Same about the other files: this is just assigning the object to itself.